spec/commands/add_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe "bundle add" do
+  before :each do
+    gemfile
+  end
+
+  context "when version number is set" do
```

Comment:
can you add a spec and feature coverage for `bundle add foo "<= 1.2.3"` (specifying a version operator in the constraint passed).